### PR TITLE
CLOUDP-134644: Fixed multi-namespace configuration for the manager

### PR DESCRIPTION
**Pull request by igor-karpukhin**
### All Submissions:

Fixed multi-namespace configuration for the manager

* [x] Have you signed our [CLA](https://www.mongodb.com/legal/contributor-agreement)?

### CODE REVIEW

File: test/e2e/integration_test.go [hunk 1]
```diff
@@ -138,3 +138,13 @@ func GetProjectIntegrationStatus(testData *model.TestDataProvider) string {
 	}
 	return ""
 }
+
+func GetProjectIntegrationStatusG(g Gomega, testData *model.TestDataProvider) string {
```
Comment (fabritsius) [inline test/e2e/integration_test.go:142]: We might no longer need this func 🤔

**Review by fabritsius — APPROVED** [review]
Looks Great 👍 
Just left a suggestion:)

File: test/e2e/integration_test.go (hunk 1 above)
Comment (igor-karpukhin) [reply test/e2e/integration_test.go:142]: Good point!

**Comment by igor-karpukhin** [comment]
related to https://github.com/mongodb/helm-charts/pull/167

**Pull request merged**